service/namespace_service.go

```diff
@@ -84,6 +85,34 @@ func (n *NamespaceService) ReconcileProjectNamespace(ctx context.Context, namesp
 				"object_kind": metricKindNamespace,
 			},
 		)
+	} else {
+		// check if the namespace has the correct labels
+		if !util.CompareLabels(nsResource.Labels, n.getResourceLabel(namespace, owner)) {
+			nsResource.Labels = n.getResourceLabel(namespace, owner)
```

Comment:
I think this should not be a replace operation. The cluster admin might have their own labels for whatever reason. We should only add our labels instead of replacing all labels on the ns.

Reply:
I agree. Fixing this